packages/web-components/src/components/ic-card/ic-card.css

```diff
@@ -6,7 +6,7 @@
 
 a {
   text-decoration: none !important;
-  color: var(--ic-architechtural-black) !important;
+  color: var(--ic-architectural-black) !important;
```

Comment:
Updating this spelling has highlighted a bug with card on hero. I'm going to raise a ticket for it to be fixed separately, so please could you remove these changes from this PR? Thank you

Reply:
Ok. 

Note: This means that "architechtural" will be added to the `.cspell.json` config file.